hal/Cargo.toml

```diff
@@ -31,45 +31,51 @@ features = ["samd21g", "samd21g-rt", "usb", "dma", "async", "rtic"]
 
 [dependencies]
 aes = "0.8.4"
-atsamd-hal-macros = {version = "0.3.0", path = "../atsamd-hal-macros"}
```

Comment:
The formatting changes make this PR hard to review - if they're necessary then please open a separate PR for them